agent/job_runner.go

```diff
@@ -587,6 +621,34 @@ func truncateEnv(l logger.Logger, env map[string]string, key string, max int) er
 	return nil
 }
 
+func (r *JobRunner)executePreBootstrapHook(hook string) (bool, error) {
+	sh, err := shell.New()
+	if err != nil {
+		return false, err
+	}
+
+	// TODO stream logging up to buildkite?
```

Comment:
I think we should not stream the logs up to buildkite.com, and possibly not even send up the process output at all. Instead just sending a static allowed/blocked message based on the exit code.

Reply:
What's the thinking here, security measure?

Reply:
Thinking about this more, I think I'd be OK with not sending the output to BK, but instead just sending it to the agents STDOUT. That way if we introduce other "agent lifecycle hooks" like this "startup", "shutdown", etc, there can be a consistent message "Bootstrap hooks are sent to BK, agent hooks are sent to STDOUT" or something.